data/docs/seldon-model-servings-tutorial.yaml

```diff
@@ -5,9 +5,8 @@ metadata:
     opendatahub.io/categories: 'AI/Machine learning,Deployment,Model serving,Python'
 spec:
   appName: seldon-deploy
-  displayName: Various types of Model Servings
+  displayName: Learning about model servings
```

Comment:
I would change to "Learning about model serving"

Reply:
LGTM